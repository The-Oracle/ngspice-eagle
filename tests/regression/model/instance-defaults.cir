* check whether .model accepts instance defaults

v1  1 0  dc=1
r1  1 0  myres

.model myres r(resistance=2k)

.control

op

if abs(i(v1)/-0.5mA - 1) > 1e-9
  echo "ERROR: check failed"
  quit 1
else
  echo "INFO: ok"
  quit 0
end

.endc
