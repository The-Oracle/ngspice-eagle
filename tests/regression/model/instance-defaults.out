
Circuit: * check whether .model accepts instance defaults

Doing analysis at TEMP = 27.000000 and TNOM = 27.000000



No. of Data Rows : 1
INFO: ok
